package.json

```diff
@@ -15,54 +15,54 @@
   "files": [
     "dist"
   ],
+  "dependencies": {
+    "@variate/engine": "^3.0.1",
+    "react": "^16.12.0"
+  },
```

Comment:
This would be a bad idea. Cause react hooks only came out in the latest version of React. Also for double loading reasons it 's better to keep react as a peer dependency.

Reply:
Was there a reason why we would need React to be loaded in the package?

Reply:
Yes, because you depend directly on it in the source: https://github.com/VariateApp/variate-react/blob/master/src/index.tsx#L1

Reply:
It would only double if the version of react does not match the version requirements, but you could make the requirements pretty loose to avoid conflicts

Reply:
Actually, when I think about it, we would have to enforce a version of react that's greater than 16.2 due to this being when createContext coming out. And we heavily rely on it. So I think this is good for now.